src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,106 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int DEFAULT_CAPACITY = 2;
+    private static final int NO_INDEX = -1;
+    private static final int SCALE = 2;
+    private int capasity;
+    private int size;
+    private Node<K, V>[] store;
+
+    public StorageImpl() {
+        this.capasity = DEFAULT_CAPACITY;
+        this.store = new Node[capasity];
```

Comment:
redundant usage of `this` keyword
```suggestion
        capasity = DEFAULT_CAPACITY;
        store = new Node[capasity];
```

Reply:
\+. OK.